saleor/graphql/account/mutations/staff.py

```diff
@@ -341,6 +214,28 @@ def perform_mutation(cls, root, info, **data):
         return results
 
 
+class CustomerRequestPasswordReset(BaseMutation):
```

Comment:
Let's have only one mutation in API to request the password reset: `requestPasswordReset(email: String): RequestPasswordReset`.